2/contrib/s2i/run

```diff
@@ -230,12 +230,19 @@ if [[ "${CONTAINER_MEMORY_IN_BYTES}" -lt $((2**40)) ]]; then
   fi
 fi
 
+FATAL_ERROR_OPTION=""
+
+if [[ "${FATAL_ERROR_LOG}" == "true" ]]; then
+    echo "Fatal error log is true. We need to set the Error File to reside in the PV."
+    FATAL_ERROR_OPTION="-XX:ErrorFile=/var/lib/jenkins/logs"
+fi
+
 if [[ -z "$JAVA_GC_OPTS" ]]; then
   # See https://developers.redhat.com/blog/2014/07/22/dude-wheres-my-paas-memory-tuning-javas-footprint-in-openshift-part-2/ .
   # The values are aggressively set with the intention of relaxing GC CPU time
   # restrictions to enable it to free as much as possible, as well as
   # encouraging the GC to free unused heap memory back to the OS.
-  JAVA_GC_OPTS="-XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90"
+  JAVA_GC_OPTS="-XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 $FATAL_ERROR_OPTION"
```

Comment:
this seems like it belongs in JENKINS_JAVA_OPTIONS, not JAVA_GC_OPTIONS.

Reply:
I can move it. The issue has an extra `-` that was throwing it off... I realized that after I moved it to `JAVA_GC_OPTIONS`